packages/cursorless-vscode-e2e/src/suite/scopes.vscode.test.ts

```diff
@@ -27,11 +27,11 @@ interface Config {
   skip?: boolean;
 }
 
-const testPaths = getTestPaths();
-
 suite("Scope test cases", async function () {
   endToEndTestSetup(this);
 
+  const testPaths = getTestPaths();
+
```

Comment:
I wasn't sure why you moved this to the root, so I moved it back and nothing broke. I think it's preferable to do big things during suite setup rather than while we're loading the file itself

Reply:
Weird I tried multiple times and the test suit just timed out before I was done getting the test paths. But now it appears to be working.